.devcontainer/init.sh

```diff
@@ -0,0 +1,12 @@
+### Create _dotnetsdk
+echo 'Creating symlink for embedded sdk...'
+ln -s /home/codespace/.dotnet/sdk/ _dotnetsdk 
+echo .6.0.100 >> _dotnetsdk/6.0.100/.6.0.100 
+ln -s /home/codespace/.dotnet/dotnet _dotnetsdk/6.0.100/dotnet
+
+### Restore 
+dotnet restore src/Runner.Listener &
```

Comment:
Are we able to just restore at the root recursively rather then having to maintain a list of all projects here